tests/test_loadDiaCatalogs.py

```diff
@@ -295,6 +296,10 @@ def setUp(self):
         self.apdb.storeDiaObjects(self.diaObjects,
                                   self.dateTime)
 
+        # These columns intentionally appear in DiaSource.yaml,
+        # but not in the APDB schema
+        self.ignoreColumns = ["filterName", "bboxSize", "isDipole"]
```

Comment:
They are specified in the schema-extra file in ap_association.

Reply:
Thanks, I figured there was someplace these not-DPDD-yet-important columns lived.

Reply:
Actually, I see `filterName` and `isDipole` in there, but where does `bboxSize` come from?